handbook/engineering/deployments.md

```diff
@@ -86,6 +86,18 @@ git push origin release
 1. Go to [renovate.json](https://github.com/sourcegraph/deploy-sourcegraph-dot-com/blob/release/renovate.json) and remove the `"extends:["default:automergeDigest"]` entry for the "Sourcegraph Docker images" group ([example](https://github.com/sourcegraph/deploy-sourcegraph-dot-com/commit/0eb16fd9e3ddfcf3a3c75ccdda0e7eddabf19c7a)).
 1. Once you have fixed the issue in the `main` branch of [sourcegraph/sourcegraph](https://github.com/sourcegraph/sourcegraph), re-enable auto-deploys by reverting your change to [renovate.json](https://github.com/sourcegraph/deploy-sourcegraph-dot-com/blob/release/renovate.json) from step 1.
 
+### Accessing sourcegraph.com database
+
+Sourcegraph.com utilizes an external HA database. You will need to connect to it directly. The easiest way to do this is through the `gcloud` cli.
+
+To connect to the production database:
+
+```
+  gcloud beta sql connect sg-cloud-732a936743 --user=sg -d sg --project sourcegraph-dev
```

Comment:
This requires cloud proxy locally?

Reply:
It technically spins up the cloud proxy locally and tunnels through it but it is all encapsulated in the command.

Reply:
I didn't have the cloud sql binary so I had to install it. There were links in the instructions though so I think it's pretty straightforward.

Reply:
Ahh ok, good to know